#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("min_radius", double_t, 0, "minium radius of torus", 0.1, 0.0, 5.0)
gen.add("max_radius", double_t, 0, "minium radius of torus", 1.0, 0.0, 5.0)
gen.add("outlier_threshold", double_t, 0, "outlier threshold", 0.01, 0.0, 1.0)
gen.add("max_iterations", int_t, 0, "max iterations", 100, 0, 1000)
gen.add("min_size", int_t, 0, "minimum number of inliers", 10, 0, 1000)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "TorusFinder"))
